src/main/scala/extracells/part/PartFluidLevelEmitter.java

```diff
@@ -54,6 +58,30 @@
 	private long currentAmount;
 	private boolean clientRedstoneOutput = false;
 
+	@Override
+	public TickingRequest getTickingRequest(final IGridNode node )
+	{
+		return new TickingRequest( TickRates.IOPort.getMin(), TickRates.IOPort.getMax(), false, true );
+	}
+
+	@Override
+	public TickRateModulation tickingRequest(final IGridNode node, final int ticksSinceLastCall )
+	{
+		if (this.fluid != null) {
+			long lastAmount = this.currentAmount;
+			updateCurrentAmount();
+			if (this.currentAmount != lastAmount) {
+				if (node != null) {
+					setActive(node.isActive());
+					getHost().markForUpdate();
+					notifyTargetBlock(getHostTile(), getSide());
```

Comment:
It would make sense to send notification not every time when fluid level changes, but only wen redstone signal changes.

Reply:
Actually onStackChange (where probably this code is from) also deserves this change, so maybe extract it to a method?